.github/workflows/buildPublish.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will run build an AS project and publish the libraries to the github package registry
+
+name: Build Publish Libraries
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*

```

Comment:
I think this setup with the two workflows makes sense - you want to auto-build and export commits from all branches except for `main`, and you want to build and publish for either `main` pushes or specific tag events. Am I understanding the intent correctly?

Reply:
That is correct. 

I think maybe a better solution would be one workflow that has steps that are conditional but I didnt want to spend time on that yet.